pallets/identity/src/lib.rs

```diff
@@ -525,6 +526,15 @@ decl_module! {
             Ok(())
         }
 
+        pub fn get_cdd_of(_origin, of: T::AccountId) -> DispatchResult {
```

Comment:
shouldn't we need to cover this in RPC call? Instead of making dispatchable make an internal function so we can use that in the next task (To create RPC). & I think we can also remove `get_my_did` dispatchable as well.

Reply:
Yeah, we can do that. I added this just to debug a function. If there's a requirement for such an RPC, we can add it via a new task.